### ISSUE TITLE
You're not authorized to access this page. Please contact the administrator. Home

### ISSUE THREAD

**Issue by mschoneman** [hkamel/sonar-auth-aad#97]
Here's what's in the log:
```
2020.03.30 18:39:04 DEBUG web[AXEHWXp07Asj8bRYAB0d][auth.event] login failure [cause|User must be authenticated][method|BASIC][provider|LOCAL|local][IP|127.0.0.1|xxx.xxx.xxx.xxx:44849, xxx.xxx.xxx.xxx:44849][login|]
2020.03.30 18:39:08 ERROR web[AXEHWXp07Asj8bRYAB0o][o.a.a.a.AadIdentityProvider] Exception:java.lang.IllegalArgumentException: authorization code is null or empty
2020.03.30 18:39:08 DEBUG web[AXEHWXp07Asj8bRYAB0o][auth.event] login failure [cause|Plugin did not call authenticate][method|OAUTH2][provider|EXTERNAL|Microsoft][IP|127.0.0.1|xxx.xxx.xxx.xxx:55824, xxx.xxx.xxx.xxx:55824][login|]
```
Working fine for one user, but not another.  Any ideas where to look?

**Comment by srvrguy** [comment]
Is there any more detail you can provide on this issue? Are there any differences in the user accounts being used? Have they both tried from the same computer, if possible?

**Comment by jmezach** [comment]
I ran into the same issue today and I tracked it down to [this article](https://blogs.aaddevsup.xyz/2019/11/aadsts650056-misconfigured-application/). I provisioned the application using the Azure CLI, which by default doesn't add the necessary permissions for this to work.

You might want to look at what's going on using your browsers developer tools. Having a look at the network tab led me to an error message.
